Gemfile.lock

```diff
@@ -34,43 +34,43 @@ GEM
       money (>= 6.0.0)
       railties (>= 3.2.0)
     erubi (1.9.0)
-    i18n (1.8.2)
+    i18n (1.8.3)
       concurrent-ruby (~> 1.0)
     json (2.3.0)
-    jwt (2.1.0)
-    jwt_signed_request (2.5.1)
-      jwt (>= 1.5.0, < 2.2.0)
+    jwt (2.2.1)
+    jwt_signed_request (2.5.4)
+      jwt (>= 1.5.0)
       rack
-    loofah (2.4.0)
+    loofah (2.6.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (1.0.0)
     mini_portile2 (2.4.0)
-    minitest (5.14.0)
-    money (6.13.6)
+    minitest (5.14.1)
+    money (6.13.7)
       i18n (>= 0.6.4, <= 2)
     nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
-    pagerduty (2.1.2)
+    pagerduty (3.0.0)
```

Comment:
### [pagerduty](http://github.com/envato/pagerduty)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 2.1.2 → 3.0.0

[_[change-log](https://github.com/envato/pagerduty/blob/v3.0.0/CHANGELOG.md), [source-code](https://github.com/envato/pagerduty/tree/v3.0.0)_]





<details>
<summary>Commits</summary>

A change of **38** commits. See the full changes on [the compare page](https://github.com/envato/pagerduty/compare/v2.1.2...v3.0.0).

These are the first 10 commits:
- (9c75f65) [Update for the latest Rubocop](https://github.com/envato/pagerduty/commit/9c75f65092bdff61d37df88523bc4d9bf7fa22fc)
- (f1faa19) [Merge pull request #54 from envato/orien/rubocop](https://github.com/envato/pagerduty/commit/f1faa19df754f6dcd9935b5cd4ca5f54b8561e04)
- (d0bec4a) [Test on Ruby 2.6](https://github.com/envato/pagerduty/commit/d0bec4a0b9350f4570d7b483094440a398234a1f)
- (09806a1) [Merge pull request #55 from envato/orien/ruby-2.6](https://github.com/envato/pagerduty/commit/09806a1956903484ff4472230a12a18fd3e91131)
- (3dda4a8) [Add project metadata to the gemspec](https://github.com/envato/pagerduty/commit/3dda4a80d95c2dc789e295aeda4e4eca181cd162)
- (4c41458) [Add shield for number of downloads](https://github.com/envato/pagerduty/commit/4c4145869ef98639c07097158e0ff63b90b77b10)
- (bdf80e2) [Merge pull request #58 from envato/readme-shields](https://github.com/envato/pagerduty/commit/bdf80e2d40150466492170c03b9727035707cd01)
- (4112528) [Merge pull request #57 from envato/gem-metadata](https://github.com/envato/pagerduty/commit/4112528bd04a28c9ca470e8f2388da00d272d124)
- (8c22957) [Add a changelog document](https://github.com/envato/pagerduty/commit/8c22957b9afd9408d8f4ba2072e3a96a0d7482cb)
- (69d9ace) [Merge pull request #59 from envato/changelog](https://github.com/envato/pagerduty/commit/69d9ace4574167363d64b1a542f04ad3b754b416)


</details>